composer.json

```diff
@@ -5,10 +5,11 @@
     "homepage": "https://github.com/clue/graph-composer",
     "license": "MIT",
     "require": {
-        "clue/graph": "~0.9.0",
+        "php": "^5.3.6 || ^7.0",
```

Comment:
guessed by travis... 
this one is required version for installing the package as dependency of other package